/buzzer_controller:
  ros__parameters:
    pin: 5
/motor_controller:
  ros__parameters:
    pins: [17, 27, 22, 18]
    esc_max: 2000
    esc_min: 1000
/imu_controller:
  ros__parameters:
    freq: 60
/gps_controller:
  ros__parameters:
    baud: 115200
    freq: 10
/ping_controller:
  ros__parameters:
    echo_pin: 6
    trig_pin: 19
    freq: 10
/**:
  ros__parameters:
    debug: true